File: abc/abc237/a.cpp
// https://atcoder.jp/contests/abc237/submissions/28901363

#include <bits/stdc++.h>
using namespace std;
int main() {
  long long n; cin >> n;
  if(pow(-2, 31) <= n && n < pow(2, 31)) cout << "Yes";
  else cout << "No";
}

File: abc/abc237/a.java
// https://atcoder.jp/contests/abc237/submissions/43760656

import java.util.Scanner;

class Main {
  static long pow(long x, long y) {
    long res = 1;
    while(y-- > 0) res *= x;
    return res;
  }

  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    long n = stdIn.nextLong();
    if(-pow(2, 31) <= n && n < pow(2, 31)) {
      System.out.println("Yes");
    } else {
      System.out.println("No");
    }
  }
}
